packages/build/src/barque.spec.ts

```diff
@@ -163,8 +165,32 @@ describe('Barque', () => {
     });
   });
 
+  describe('LATEST_CURATOR', () => {
+    it('can be downloaded', async() => {
+      const response = await fetch(LATEST_CURATOR, {
+        method: 'HEAD'
+      });
+
+      expect(response.ok).to.be.true;
+    });
+  });
+
   describe('.extractLatestCurator', () => {
-    it('extractss latest curator to tmp directory', async() => {
```

Comment:
this used to try and download the file for real, which timeouts with a slow connection,  and .. unfortunately my connection is slow :)